OpenOversight/tests/routes/test_officer_and_department.py

```diff
@@ -1830,8 +1837,14 @@ def test_ac_can_upload_photos_of_dept_officers(
         officer = department.officers[4]
         officer_face_count = len(officer.face)
 
-        crop_mock = MagicMock(return_value=Image.query.first())
-        upload_mock = MagicMock(return_value=Image.query.first())
+        # Filter out images that the officer is already tagged in
+        officer_faces = Face.query.filter_by(officer_id=officer.id).all()
+        image = Image.query.filter(
+            Image.id.not_in([face.img_id for face in officer_faces])
+        ).first()
+
+        crop_mock = MagicMock(return_value=image)
+        upload_mock = MagicMock(return_value=image)
```

Comment:
Should we make this a test util function? I recognize that there's already a ton of duplication in the tests so feel free to ignore 😂